### PR TITLE
Simple viewer example

### PR DESCRIPTION
**Pull request by hackermd**
Finally added the viewer example

### CODE REVIEW

File: examples/simple_viewer/index.html [hunk 1]
```diff
@@ -0,0 +1,517 @@
+<!DOCTYPE html>
+<html>
+
+  <head>
+
+    <title>DICOM WSI Viewer</title>
+    <meta charset="utf-8" />
+    <link type="text/css" rel="stylesheet" href="styles/main.css">
+    <script type="text/javascript" src="https://unpkg.com/dicomweb-client"></script>
```
Comment (pieper) [inline examples/simple_viewer/index.html]: @swederik said he was having trouble with unpkg refreshing unless he included the version.  See:  https://github.com/dcmjs-org/dcmjs-examples/commit/5367df2ba36131ee8f7f9df9de054b58da89ba3b

File: examples/simple_viewer/index.html [hunk 2]
```diff
@@ -0,0 +1,517 @@
...
+        const studySearchOptions = {
+          queryParams: {
+            ModalitiesInStudy: 'SM'
+          }
+        }
+        client.searchForStudies(studySearchOptions).then((studies) => {
+          const studyList = getDomElement("study-list", removeChildren=true);
+          const attributes = [
+            "00200010", "00080020", "00080030", "00081030",
+            "00100020", "00100010", "00100040", "00100030"
```
Comment (pieper) [inline examples/simple_viewer/index.html]: Sure would be nicer to use the dictionary here and avoid the hex...

File: examples/simple_viewer/index.html (hunk 1 above)
Comment (hackermd) [reply examples/simple_viewer/index.html]: Addressed with 7789763f1edee6a1a49353d70951488c6478a276

**Comment by pieper** [comment]
👍

**Pull request merged**